### PR TITLE
adding ca certs for FR unleash

### PR DESCRIPTION
**Pull request by suzana-nesic**
Currently running into the error in FR:
```
"oauthError": {
    "code": "UNABLE_TO_VERIFY_LEAF_SIGNATURE"
  }
```

Following other FR patterns to provide the CA so it can be a trusted certificate

### CODE REVIEW

**Comment by cubismod** [comment]
Here's an example of us using this pattern for git-sync: https://github.com/app-sre/git-partition-sync-consumer/blob/master/openshift/consumer.yaml#L96

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+generate:
+	@helm lint helm/unleash
+	@helm template helm/unleash -n unleash -f helm/unleash/values-commercial.yaml > openshift/unleash.yaml
+	@helm template helm/unleash -n unleash -f helm/unleash/values-fedramp.yaml > openshift/unleash-fedramp.yaml
```
Comment (janboll) [inline Makefile:4]: missing EOL

File: openshift/unleash-fedramp.yaml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  value: ""
+- name: editor_roles
+  value: ""
+- name: viewer_roles
+  value: ""
+- name: issuer_name
+  value: letsencrypt-prod-http
+- name: INTERNAL_GIT_CA_PATH
+  description: 'path to certificate file to add in git config trust'
+  value: '/etc/pki/ca-trust/source/anchors/ca.crt'
```
Comment (janboll) [inline openshift/unleash-fedramp.yaml:226]: missing EOL

**Review by cubismod — APPROVED** [review]
LGTM!

**Pull request merged**